src/plugins/index_pattern_management/public/management_app/mount_management_section.tsx

```diff
@@ -0,0 +1,191 @@
+} from '../components';
+import { IndexPatternManagementStartDependencies, IndexPatternManagementStart } from '../plugin';
+
+const appTitle = i18n.translate('indexPatternManagement.indexPattern.sectionsHeader', {
+  defaultMessage: 'Index Patterns',
+});
+const crumb = [{ text: appTitle }];
+
+const readOnlyBadge = {
+  text: i18n.translate('indexPatternManagement.indexPatterns.badge.readOnly.text', {
+    defaultMessage: 'Read only',
+  }),
+  tooltip: i18n.translate('indexPatternManagement.indexPatterns.badge.readOnly.tooltip', {
+    defaultMessage: 'Unable to save index patterns',
+  }),
+  iconType: 'glasses',
+};
+
+export async function mountManagementSection(
+  getStartServices: StartServicesAccessor<IndexPatternManagementStartDependencies>,
+  params: ManagementAppMountParams
+) {
+  params.setBreadcrumbs(crumb);
+  const [
+    { chrome, application, savedObjects, uiSettings, notifications, overlays, http, docLinks },
+    { data },
+    indexPatternManagementStart,
+  ] = await getStartServices();
+  const defaultIndex = uiSettings.get('defaultIndex');
+  const canSave = application.capabilities.indexPatterns.save as boolean;
+
+  if (!canSave) {
+    chrome.setBadge(readOnlyBadge);
+  }
+
+  const indexPatterns =
+    (await savedObjects.client
+      .find<IIndexPattern>({
+        type: 'index-pattern',
+        fields: ['title', 'type'],
+        perPage: 10000,
+      })
+      .then(response =>
+        response.savedObjects
+          .map(pattern => {
+            const id = pattern.id;
+            const title = pattern.get('title');
+            const isDefault = defaultIndex === id;
+
+            const tags = (indexPatternManagementStart as IndexPatternManagementStart).list.getIndexPatternTags(
+              pattern,
+              isDefault
+            );
+
+            return {
+              id,
+              title,
+              url: '',
+              // url: kbnUrl.eval('#/management/kibana/indexPatterns/create{{id}}', { id }),
+              // active: $scope.editingId === id,
+              default: isDefault,
+              tags,
+              // the prepending of 0 at the default pattern takes care of prioritization
+              // so the sorting will but the default index on top
+              // or on bottom of a the table
+              sort: `${isDefault ? '0' : '1'}${title}`,
+            };
+          })
+          .sort((a, b) => {
+            if (a.sort < b.sort) {
+              return -1;
+            } else if (a.sort > b.sort) {
+              return 1;
+            } else {
+              return 0;
+            }
+          })
+      )) || [];
+
+  // todo - need new path to create index pattern
+  // todo navigate to items
+  // todo path w/o slash
+  //
+  // indexPattern, indexPatterns
+
+  // data.indexPatterns.get()
+
+  ReactDOM.render(
+    <I18nProvider>
+      <HashRouter basename={params.basePath}>
+        <Switch>
+          <Route path={['/create']}>
```

Comment:
We should wrap each route using the next constuction (you can find it here: \src\plugins\saved_objects_management\public\management_section\mount_section.tsx)  :
```
<RedirectToHomeIfUnauthorized capabilities={capabilities}>
              <Suspense fallback={<EuiLoadingSpinner />}>
```

For this, we should create component RedirectToHomeIfUnauthorized like in saved_objects_management. 
It helps to fix the problem with privileges.

Reply:
Thanks for pointing out this technique. I think user permissions can be handled via the management app but I need to verify this.

Reply:
Sorry, I checked and found out that if we use the next construction in start block we are able to not use RedirectToHomeIfUnauthorized : 
```
if (!core.application.capabilities.management.kibana.index_patterns) {
      this.managementApp!.disable();
    }
```